RadarSDK/Include/Radar.h

```diff
@@ -649,13 +662,22 @@ typedef void (^_Nonnull RadarLogConversionCompletionHandler)(RadarStatus status,
             completionHandler:(RadarLogConversionCompletionHandler)completionHandler NS_SWIFT_NAME(logConversion(name:revenue:metadata:completionHandler:));
 
 /**
-logConversionWithNotification
+Logs a conversion with a notification
```

Comment:
@KennyHuRadar @lmeier Let's try to be more consistent with comments/docs. Other functions are doc'd like `Manually updates the user's location.`, with a leading space and a period at the end. So maybe change `Logs a conversion with a notification` to ` Logs a conversion with a notification.`